File: Admin_p/Admin_p.Web/Modules/ServerTypes/Identity/AspNetUserColumns.ts
import { ColumnsBase, fieldsProxy } from '@serenity-is/corelib';
import { Column } from '@serenity-is/sleekgrid';
import { AspNetUserRow } from './AspNetUserRow';

export interface AspNetUserColumns {
    Id: Column<AspNetUserRow>;
    Name: Column<AspNetUserRow>;
    NormalizedName: Column<AspNetUserRow>;
    ConcurrencyStamp: Column<AspNetUserRow>;
}

export class AspNetUserColumns extends ColumnsBase<AspNetUserRow> {
    static readonly columnsKey = 'Identity.AspNetUser';
    static readonly Fields = fieldsProxy<AspNetUserColumns>();
}

File: Admin_p/Admin_p.Web/Modules/Identity/AspNetUser/AspNetUserColumns.cs
using Serenity.ComponentModel;
using System.ComponentModel;

namespace Admin_p.Identity.Columns;

[ColumnsScript("Identity.AspNetUser")]
[BasedOnRow(typeof(AspNetUserRow), CheckNames = true)]
public class AspNetUserColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight, EditLink]
    public string Id { get; set; }
    public string Name { get; set; }
    public string NormalizedName { get; set; }
    public string ConcurrencyStamp { get; set; }
}